index.html

```diff
@@ -119,15 +119,20 @@
       </h2>
       <p>
         This specification describes an API that allows <a>user agents</a>
-        (e.g., browsers) to act as an intermediary between three systems in
-        every transaction: the merchant (e.g., an online web store), the buyer,
-        represented by the <a>user agent</a> (e.g., the user buying from the
-        online web store), and the <dfn data-lt="payment methods">payment
-        method</dfn> (e.g., credit card). Information necessary to process and
-        confirm a transaction is passed between the <a>payment method</a> and
-        the merchant via the <a>user agent</a> with the buyer confirming and
-        authorizing as necessary across the flow.
+        (e.g., browsers) to act as an intermediary between three parties in a
+        transaction:
       </p>
+      <ul>
+        <li>the payee, such as a merchant with an online store,
```

Comment:
I'd still like this to be:

 * The payee: bla bla bla. 
 * The payer: bla bla bla.
 * The provider: bla bla bla.